client/client_test.go

```diff
@@ -551,6 +560,89 @@ func withToken(t *testing.T, asUser *client.Client) *client.Client {
 	return asUser
 }
 
+var tearDownResource = func(pool *dockertest.Pool, r *dockertest.Resource) error {
+	var err error
+	if pool == nil {
+		pool, err = setupDockerPool()
+		if err != nil {
+			return err
+		}
+	}
+	err = pool.Purge(r)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func setupCoreInstance(pool *dockertest.Pool, baseURL string, token types.Token) (*dockertest.Resource, error) {
+	var err error
+	if pool == nil {
+		pool, err = setupDockerPool()
+		if err != nil {
+			return nil, err
+		}
+	}
+
+	// Determine kubernetes config location
+	if kubeConfig == "" {
+		kubeConfig = filepath.Join(os.Getenv("HOME"), "/.kube/config")
```

Comment:
There is an `os.UserHomeDir()` https://pkg.go.dev/os#UserHomeDir you can use here.